service/templates/projects.html

```diff
@@ -282,7 +283,48 @@ <h3>Alerts</h3>
             </div>
           </div>
         </div>
-        <br>
+
+        <div class="variables">
```

Comment:
Should we hide this section for now so users don't get confused?

Reply:
Nah.